data_relay.py

```diff
@@ -41,7 +41,6 @@
         ('B', 'gpsStatus'),
         ('B', 'batteryLevel'),
         ('B', 'waypointCount'),
-	('B', 'airspeed'),
```

Comment:
Correction, the data struct is 2 byte packed, the payload can be arbitrarily sized, but the network code assumes the size of payload is the size of the struct. This means for odd-byte structs there is a byte of padding (the commented line following this one).